docs/docs/timelock-encryption.md

```diff
@@ -0,0 +1,119 @@
+
+
+## ⚠️ Security assumptions ⚠️
+
+While our timelock scheme has been formally audited by reknowned security firm [Kudelski Security](https://kudelskisecurity.com)[1], as with any cryptographic scheme, it's important to understand its limitations before you go and bet the farm on it! This is not an exhaustive list, and you ought to also read the [drand security model](https://drand.love/docs/security-model/), but it's a good place to start.
+
+- **there have never been a threshold of malicious nodes in the drand network**
+
+If there were, at any time, a threshold number of malicious nodes in the network, they could cooperatively generate all future random values and use them to decrypt arbitrary future timelock ciphertexts. Drand randomness, while unpredictable, is entirely deterministic, even after the resharing of keys.
+That said, `fastnet` (our new network for timelock encryption) was started with 16 different participants running 23 nodes, so we're confident a threshold number of them have not been compromised!
+
+- **neither the scheme, nor drand are quantum resistant**
+
+The cryptography we employ do not use quantum resistant algorithms. If you timelock encrypt something for 1000 years and a viable quantum computer becomes available, anybody can decrypt it. Right now there are no widespread quantum resistant schemes for threshold cryptography, but we're keeping an eye out for the future!
+To that end, consider it relatively safe to timelock encrypt something for 5-20 years depending on how bullish you are on quantum computing.
+
+- **if the League of Entropy shuts down, members will delete their keys**
+
+The world is unpredictable (just like drand randomness... heh), and it's possible that the League of Entropy will all hang up their coats at some point in the future. Should that happen, members would have two choices with their private keys: release them to the world or delete them entirely.
+The former would mean that ciphertexts created for some time after the cessation of the network would be decryptable to everybody. The latter would mean that ciphertexts created for some time after the cessation of the network would be unencryptable forever (/until quantum computers can break them).
+In the interests of privacy, we felt the latter option was preferable. That said, if you encrypt the private key to your Filecoin fortune to stop yourself from spending it now and the network stops... you're going to have a bad time.
+
+[1] their report is available in timelocked form [here](https://research.kudelskisecurity.com/2023/01/09/announcing-a-timelocked-responsible-disclosure/), although it was encrypted using the drand testnet, so you will need to use the [tlock go library](https://github.com/drand/tlock) to decrypt it by passing in the testnet details. All the security issues they identified have since been remediated! You can find remediation details in the github issues of the relevant repos.
```

Comment:
It's not. Actually that's just the "date" issue, not the whole report.

Reply:
Ah hmm, did they timelock the whole report anywhere?!

Reply:
Not that I know of, and we are yet to do a public blog post and release of it I think. Let's coordinate with them

Reply:
Grand, I'll modify this section to say that they just timelocked part of it